client/state/reader/follows/selectors/is-following.js

```diff
@@ -7,7 +7,7 @@ import { find } from 'lodash';
  * Internal dependencies
  */
 import { prepareComparableUrl } from 'state/reader/follows/utils';
-import getReaderAliasedFollowFeedUrl from 'state/selectors/get-reader-aliased-follow-feed-url';
+import getReaderAliasedFollowFeedUrl from './get-reader-aliased-follow-feed-url';
```

Comment:
I would generally advise against the relative paths when possible. That makes it easier and less error-prone when moving files here and there between directories.